app/uk/gov/hmrc/partnershipidentificationfrontend/testonly/forms/TestCreateJourneyForm.scala

```diff
@@ -19,19 +19,18 @@ package uk.gov.hmrc.partnershipidentificationfrontend.testonly.forms
 import play.api.data.Form
 import play.api.data.Forms.{mapping, text}
 import play.api.data.validation.Constraint
+import uk.gov.hmrc.partnershipidentificationfrontend.forms.utils.MappingUtil.optText
 import uk.gov.hmrc.partnershipidentificationfrontend.forms.utils.ValidationHelper.validate
 import uk.gov.hmrc.partnershipidentificationfrontend.models.{JourneyConfig, PageConfig}
-import uk.gov.hmrc.partnershipidentificationfrontend.forms.utils.MappingUtil.optText
 
 
 object TestCreateJourneyForm {
 
-  var continueUrl = "continueUrl"
```

Comment:
var?!

Reply:
I'm not sure why this was a var previously, I'll add this to the cleanup commit